### PR TITLE
Update renderArea VUIDs

### PR DESCRIPTION
**Pull request by ziga-lunarg**

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -10880,16 +10880,31 @@ bool CoreChecks::ValidatePrimaryCommandBuffer(const CMD_BUFFER_STATE *pCB, char
...
-        (pRenderPassBegin->renderArea.offset.y + pRenderPassBegin->renderArea.extent.height) > framebuffer_info->height) {
-        skip |= static_cast<bool>(LogError(
-            pRenderPassBegin->renderPass, kVUID_Core_DrawState_InvalidRenderArea,
-            "Cannot execute a render pass with renderArea not within the bound of the framebuffer. RenderArea: x %d, y %d, width "
-            "%d, height %d. Framebuffer: width %d, height %d.",
-            pRenderPassBegin->renderArea.offset.x, pRenderPassBegin->renderArea.offset.y, pRenderPassBegin->renderArea.extent.width,
-            pRenderPassBegin->renderArea.extent.height, framebuffer_info->width, framebuffer_info->height));
+    if (pRenderPassBegin->renderArea.offset.x < 0) {
+        skip |= LogError(pRenderPassBegin->renderPass, "VUID-VkRenderPassBeginInfo-renderArea-02846",
+                         "Cannot execute a render pass with renderArea not within the bound of the framebuffer, "
```
Comment (sfricke-samsung) [inline layers/core_validation.cpp:10885]: in `ValidateCmdBeginRenderPass` can you pass `function_name` and make it

```
skip |= VerifyRenderAreaBounds(pRenderPassBegin, function_name);
```

and print the function at start of error message, it is very helpful to know which function triggered the message quickly

File: layers/core_validation.cpp [hunk 2]
```diff
@@ -10880,16 +10880,31 @@ bool CoreChecks::ValidatePrimaryCommandBuffer(const CMD_BUFFER_STATE *pCB, char
...
-        pRenderPassBegin->renderArea.offset.y < 0 ||
-        (pRenderPassBegin->renderArea.offset.y + pRenderPassBegin->renderArea.extent.height) > framebuffer_info->height) {
-        skip |= static_cast<bool>(LogError(
-            pRenderPassBegin->renderPass, kVUID_Core_DrawState_InvalidRenderArea,
-            "Cannot execute a render pass with renderArea not within the bound of the framebuffer. RenderArea: x %d, y %d, width "
-            "%d, height %d. Framebuffer: width %d, height %d.",
-            pRenderPassBegin->renderArea.offset.x, pRenderPassBegin->renderArea.offset.y, pRenderPassBegin->renderArea.extent.width,
-            pRenderPassBegin->renderArea.extent.height, framebuffer_info->width, framebuffer_info->height));
+    if (pRenderPassBegin->renderArea.offset.x < 0) {
+        skip |= LogError(pRenderPassBegin->renderPass, "VUID-VkRenderPassBeginInfo-renderArea-02846",
```
Comment (sfricke-samsung) [inline layers/core_validation.cpp:10884]: depending if `VK_KHR_device_group` is set (and `deviceRenderAreaCount` is not being used) these VUID are also

- `VUID-VkRenderPassBeginInfo-pNext-02850`
- `VUID-VkRenderPassBeginInfo-pNext-02851`
- `VUID-VkRenderPassBeginInfo-pNext-02852`
- `VUID-VkRenderPassBeginInfo-pNext-02853`

**Review by ncesario-lunarg — COMMENTED** [review]
I thinks looks good once @sfricke-samsung's comments have been addressed.

File: layers/core_validation.cpp (hunk 1 above)
Comment (ziga-lunarg) [reply layers/core_validation.cpp:10885]: Good point, thanks

**Review by ncesario-lunarg — APPROVED** [review]
LGTM!

**Review by ncesario-lunarg — APPROVED** [review]

**Pull request merged**